source/jobs/index.md

```diff
@@ -2,32 +2,32 @@
 layout: formatted-text
 title: Join the team
 ---
-<p class="intro-copy">There’ll soon be more seats around Convivio’s big table. It’s been a busy summer already and we’re planning to expand our team.</p>
+<p class="intro-copy">We are not currently hiring for any positions.</p>
 
-We prefer building relationships to the traditional ‘CV out of the blue’ approach — so we’d like to start hearing from people now who’d be interested in joining us in the coming months.
+We're always looking to hear from people who would be interested in working with us in the future. Drop us a line at <a href="mailto:jobs@weareconvivio.com">jobs@weareconvivio.com</a>
 
-If you’re looking for a new challenge in digital, then Convivio could be the perfect next move for your career.
+## What it's like to work at Convivio
 
-## What are we like?
+At Convivio we're not just remote, [we're free-range.](https://blog.weareconvivio.com/free-range-working-an-introduction-27eb178db97c) We don't have a central office and we all work from home. We encourage flexible working. [Read more about working from home at Convivio.](/jobs/working-from-home)
```

Comment:
The `/jobs/working-from-home` page doesn't yet exist, so this can't be merged until that page is added.